app/routes/quickstart/code.tsx

```diff
@@ -10,14 +10,18 @@ import Tabs from '~/components/Tabs'
 import { ClientSampleCode } from '~/components/ClientSampleCode'
 import { Form, useLoaderData } from '@remix-run/react'
 import type { DataFunctionArgs } from '@remix-run/node'
-import { ClientCredentialVendingMachine } from '../client_credentials.server'
+import { ClientCredentialVendingMachine } from '../user/client_credentials.server'
 import { Button } from '@trussworks/react-uswds'
+import { NoticeTypes } from '~/components/NoticeTypeCheckboxes'
 
 export async function loader({ request }: DataFunctionArgs) {
   const { clientId, noticeFormat, ...rest } = Object.fromEntries(
     new URL(request.url).searchParams
   )
-  const noticeTypes = Object.keys(rest)
+  const noticeTypes = Object.keys(rest).filter(
+    (noticeType) => Object.keys(NoticeTypes).indexOf(noticeType) == -1
```

Comment:
Is this filtering necessary? Are there any entries from the form that are not notice types?

`Object.keys(NoticeTypes)` will not be a list of notice types; it will be a list of the top-level categories of the notice types (like `'Fermi'`, `'Swift'`, etc.).

Reply:
That is correct, if you select them from the parent, then we get the parent in the resulting code
![Screen Shot 2022-07-06 at 11 39 46 AM](https://user-images.githubusercontent.com/11023698/177590001-29de185b-dcd4-4d4d-9e51-24866f114ed9.png)

Reply:
resolved in https://github.com/tachgsfc/gcn.nasa.gov/pull/136